agent/bench-scripts/tests/pbench-trafficgen/test-50.txt

```diff
@@ -0,0 +1,105 @@
++++ Running test-50 pbench-trafficgen
+tool trigger strings for start: "Starting binary-search" and for stop: "Finished binary-search" are now registered for tool group: "default"
+found IOMMU option, continuing
+found dpdk-tools, continuing
+Total number of benchmark iterations: 1
+cp: cannot stat '/var/tmp/pbench-test-bench/pbench-agent/tmp/trex_cfg.yaml': No such file or directory
```

Comment:
Seems like we are facing something unusual here.

Reply:
This was missing a command line option.